test/regressions/index.js

```diff
@@ -69,7 +69,12 @@ function excludeDemoFixture(suite, name, path) {
 }
 
 // Also use some of the demos to avoid code duplication.
-const importDemos = require.context('docs/data', true, /\.js$/, 'lazy');
+const importDemos = require.context(
+  'docs/src/app/(public)/(content)/react/components',
+  true,
+  /\.tsx$/,
```

Comment:
Have we decided to have all the demos in the current dir structure going forward instead of `docs/data`?
And also no more `.jsx` version of the demos?
@vladmoroz @michaldudak

Reply:
Yeah, I don't think the JSX version is needed. It's virtually the same almost in all cases.

No plans for `docs/data` other than to migrate its contents to app directory and delete it 🙂

Also while we are here
- We might have demos in the `utils` folder too
- Also as a rule I've also been putting demos into the `demos` folder to make search easier in VS Code, maybe you can rely on this here. We can then safely add components beside individual pages if we ever need to.

Reply:
> No plans for docs/data other than to migrate its contents to app directory and delete it 🙂

We can leave this for the end (maybe even after the release, as it would break some things in the infra, it may require some mork work).

Reply:
I've set the path to `docs/src/app/(public)/(content)/react` to keep it simple, even in the old docs structure the contents were 100% demos anyway